### PR TITLE
feat: internal table support (contributors)

### PR DESCRIPTION
**Pull request by xxchan**
close https://github.com/risinglightdb/risinglight/issues/99

```sql
> \dt
+---+----------+---+------------+---+--------------+
| 0 | postgres | 1 | pg_catalog | 0 | contributors |
+---+----------+---+------------+---+--------------+
> select * from pg_catalog.contributors
+--------------+
| name         |
+--------------+
| Alex Chi     |
| Alissa Tung  |
| ArkBriar     |
| Chojan Shang |
| Frank Ma     |
| Lu Di        |
| Mingj        |
| MingjiHan    |
| Runji Wang   |
| Tab Zhang    |
| TennyZhuang  |
| Ting Sun     |
| Yingjun Wu   |
| Yuhui Liu    |
| Yujia Qiao   |
| Zehao Wei    |
| kutori       |
| likg227      |
| nanderstabel |
| stonepage    |
| wzl          |
| xxchan       |
| xyz          |
| 庄天翼       |
|              |
+--------------+
```

```sql
> select count(*) from pg_catalog.contributors
+-------+
| count |
+-------+
| 25    |
+-------+
```

### CODE REVIEW

**Comment by xxchan** [comment]
Btw, the generate contributors list contains duplicate names 😅

**Review by skyzh — COMMENTED** [review]
LGTM! But I prefer not to add too much to our `build.rs`... Now every time we checkout to a new branch or add a new commit, we will trigger a full rebuild...

I think it's better to store contributor list to a JSON, and update it on every release.

**Comment by xxchan** [comment]
> Now every time we checkout to a new branch or add a new commit, we will trigger a full rebuild...

Yes I also have this concern😓😓😓

> I think it's better to store contributor list to a JSON, and update it on every release.

Maybe generated by a pre-release script? (and now we can use this api https://api.github.com/repos/risinglightdb/risinglight/contributors)
Is there something like this? (I don't know much about how to release)

**Comment by skyzh** [comment]
Maybe we can take a look at how https://github.com/datafuselabs/databend update their contributor table?

**Comment by xxchan** [comment]
I actually looked. They use build.rs 😸

File: build.rs [hunk 1]
```diff
@@ -4,8 +4,30 @@ use std::env;
...
+        .arg("HEAD")
+        .arg("-s")
+        .output()
+        .unwrap();
+    let authors: String = String::from_utf8(output.stdout)
+        .unwrap()
+        .lines()
+        .map(|s| {
+            let mut s = s[7..].to_owned();
+            s.push(',');
```
Comment (xxchan) [inline build.rs:21]: buggy, there's a trailing ',' at the end 🤪🤪🤪

**Review by skyzh — APPROVED** [review]

**Pull request merged**